client/src/components/QA/QAItem.jsx

```diff
@@ -17,8 +23,85 @@ export default function QAItem({ question, allAnswers }) {
       totalAsToRender.push(arrayOfAnswers[i]);
     }
   }
-  // console.log(totalAsToRender);
-  // console.log(moreAsClicked && totalAsToRender[0] !== undefined);
+
+  const handleQuestionHelpful = (category, ID) => {
+    axios({
+      method: 'PUT',
+      url: `http://localhost:3000/qa/questions/${ID}/${category}`,
```

Comment:
We can omit the 'localhost' portion. You can change it to just the endpoint like `qa/questions/${ID}/${category}`